doc/ch-convert_desc.rst

```diff
@@ -0,0 +1,259 @@
+
+Many of the copy algorithms have shortcuts available, which :code:`ch-convert`
+knows about, so you should typically invoke it only once per workflow, asking
+for the final format you actually need. Some copies require temporary storage
+equal to the uncompressed size of the image.
+
+
+Arguments
+=========
+
+  :code:`IN`
+    Descriptor for the input image. In the case of builder storage, this is an
+    image reference; otherwise, a filesystem path.
+
+  :code:`OUT`
+    Descriptor for the output image.
+
+  :code:`OUT_ARG ...`
+    (Valid only for :code:`tar` and :code:`squash` output formats.) Additional
+    arguments passed unchanged to the packing command.
+
+  :code:`-b`, :code:`--builder BUILDER`
+    Use image builder :code:`BUILDER`. Default: :code:`$CH_BUILDER` if set;
+    otherwise guessed (see :code:`ch-build(1)`).
+
+  :code:`-c`, :code:`--out-compress (none|gzip|xz)`
+    (Valid only for :code:`tar` and :code:`squash` output formats.)
+    Compression algorithm for archive file. Default: :code:`$CH_COMPRESSION`
+    if set; otherwise :code:`gzip`.
+
+  :code:`-h`, :code:`--help`
+    Print help and exit.
+
+  :code:`-i`, :code:`--in-fmt (builder|dir|tar|squash)`
+    Format of the input image. If omitted, inferred as described below.
+
+  :code:`--no-clobber`
+    Complain and fail if :code:`OUT` already exists, rather than replacing it.
+
+  :code:`-o`, :code:`--out-fmt (builder|dir|tar|squash)`
+    Format of the output image; inferred if omitted.
+
+  :code:`-s`, :code:`--builder-storage ARG`
+    Some builders take an argument specifying where to find their storage,
+    e.g. :code:`ch-grow --storage=/path`. If specified, pass :code:`ARG` to
+    the builder. Default: :code:`$CH_GROW_STORAGE` if set; otherwise, see
+    :code:`ch-grow(1)`. (Currently only implemented for :code:`ch-grow(1)`.)
+
+  :code:`--tmp DIR`
+    Path to temporary directory. Storage needed varies between zero and the
+    uncompressed size of the image, depending on the input and output formats.
+    Default: :code:`$TMP` if specified; otherwise :code:`/tmp`.
+
+  :code:`-v`, :code:`--verbose`
+    Print extra chatter.
+
+
+Format and filename inference
+=============================
+
+:code:`ch-convert` tries to save typing by guessing formats and filenames when
+they are reasonably clear.
+
+Format inference is done for both :code:`IN` and :code:`OUT`. The algorithm
+tries to match the value against the following globs in this order. Paths need
+not exist in the filesystem.
+
+  1. :code:`*.sqfs`, :code:`*.squash`, :code:`*.squashfs`: SquashFS.
+
+  2. :code:`*.tar`, :code:`*.t?z`, :code:`*.tar.??`, :code:`*.tar.??`: Tarball.
+
+  3. :code:`/*`, :code:`./*` (i.e., absolute path or relative path with
+     explicit dot): Directory.
+
+  4. :code:`*:*` (i.e., containing a colon): Builder storage.
+
+  5. Otherwise: No format inference.
+
+A notable consequence of these rules is than a simple image reference such as
```

Comment:
I think "than" was a typo? Did you mean "that"?

Reply:
Yes; thank you.